launcher_go/v2/cli_runtime_test.go

```diff
@@ -0,0 +1,191 @@
+package main_test
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"bytes"
+	"context"
+	ddocker "github.com/discourse/discourse_docker/launcher_go/v2"
+	. "github.com/discourse/discourse_docker/launcher_go/v2/test_utils"
+	"github.com/discourse/discourse_docker/launcher_go/v2/utils"
+	"os"
+)
+
+var _ = Describe("Runtime", func() {
+	var testDir string
+	var out *bytes.Buffer
+	var cli *ddocker.Cli
+	var ctx context.Context
+
+	BeforeEach(func() {
+		utils.DockerPath = "docker"
+		out = &bytes.Buffer{}
+		utils.Out = out
+		testDir, _ = os.MkdirTemp("", "ddocker-test")
+
+		ctx = context.Background()
+
+		cli = &ddocker.Cli{
+			ConfDir:      "./test/containers",
+			TemplatesDir: "./test",
+			BuildDir:     testDir,
+		}
+		utils.CmdRunner = CreateNewFakeCmdRunner()
+	})
+	AfterEach(func() {
+		os.RemoveAll(testDir)
+	})
+
+	Context("When running run commands", func() {
+
+		var checkStartCmd = func() {
+			cmd := GetLastCommand()
+			Expect(cmd.String()).To(ContainSubstring("docker ps -q --filter name=test"))
+			cmd = GetLastCommand()
+			Expect(cmd.String()).To(ContainSubstring("docker ps -a -q --filter name=test"))
+			cmd = GetLastCommand()
+			Expect(cmd.String()).To(ContainSubstring("docker run"))
+			Expect(cmd.String()).To(ContainSubstring("-d"))
+			Expect(cmd.String()).To(ContainSubstring("--name test local_discourse/test /sbin/boot"))
```

Comment:
I printed out the full command here and noticed that we are still passing `--ENV` through to `docker run`. My impression is that this is not necessary because the `ENV`s are already baked into the image?